pushmsixscript/src/pushmsixscript/script.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+""" Push msix Script main script
+"""
+import logging
+import os
+
+from scriptworker import client
```

Comment:
Ideally we'll use [`scriptworker_client`](https://github.com/mozilla-releng/scriptworker-scripts/tree/master/scriptworker_client) instead of importing scriptworker. There are still some scripts that do the latter, but the former is encouraged, and if we are ever able to prioritize it, we'll complete that transition.